homeassistant/components/frontend/__init__.py

```diff
@@ -579,19 +579,24 @@ def _is_latest(js_option, request):
     if js_option != 'auto':
         return js_option == 'latest'
 
-    from user_agents import parse
-    useragent = parse(request.headers.get('User-Agent'))
-
-    # on iOS every browser is a Safari which we support from version 10.
-    if useragent.os.family == 'iOS':
-        return useragent.os.version[0] >= 10
-
-    family_min_version = {
-        'Chrome': 50,   # Probably can reduce this
-        'Firefox': 43,  # Array.protopype.includes added in 43
-        'Opera': 40,    # Probably can reduce this
-        'Edge': 14,     # Array.protopype.includes added in 14
-        'Safari': 10,   # many features not supported by 9
-    }
-    version = family_min_version.get(useragent.browser.family)
-    return version and useragent.browser.version[0] >= version
+    useragent = request.headers.get('User-Agent')
```

Comment:
Just return here instead of indenting the rest code.

Reply:
k

Reply:
I changed it to this to make it obvious in terms of dependence. Just returning, while avoiding a tiny bit of nesting, does not at first glance.

Reply:
Also, your last resort should *always* go at the end imo, unless I'm misunderstanding what you asked, which is totally possible!

Reply:
To follow that up, I'll happily change it if that's what you prefer, but I need to ask: is that really what you want?